src/main/java/pl/ug/virtualofficebackend/domain/entity/Decoration.java

```diff
@@ -0,0 +1,99 @@
+package pl.ug.virtualofficebackend.domain.entity;
+
+import javax.persistence.*;
+
+@Entity(name = "decoration")
+public class Decoration {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String name;
+
+    /*
+        Na ten moment start i koniec przedmiotu
+        wszystko bedzie traktowane jako kwadrat
+
+        np. doniczka zaczyna sie w (0,0) i konczy w (1,1)
+
+        North/South/West/East

```

Comment:
No plz, zrób z tego enuma zamiast Stringa

Reply:
no jak tak ładnie prosisz to już zrobie jak wróce do domku